src/transformers/generation_stopping_criteria.py

```diff
@@ -0,0 +1,30 @@
+import time
+from typing import Optional
+
+import torch
+
+
+class StoppingCriteria:
```

Comment:
I think I'd prefer to make this an abstract `ABC` class, just like it's done for the `LogitsProcessor` class

Reply:
I don't really understand why you add `ABC`, because you don't use `@abstractmethod` (or others).

What difference does it make ?